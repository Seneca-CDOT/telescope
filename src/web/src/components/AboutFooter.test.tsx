import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

// Auto-hoisting of jest.mock calls is currently not working https://github.com/vercel/next.js/discussions/31152
// So we have to import AboutFooter after jest.mock
jest.mock(
  './GitHubContributorCard',
  () =>
    function MockGitHubCard() {
      return <div>GitHubContributorCard</div>;
    }
);

it('renders correctly', () => {
  // eslint-disable-next-line global-require
  const AboutFooter = require('./AboutFooter').default;
  const { asFragment } = render(<AboutFooter />);
  expect(asFragment()).toMatchSnapshot();
});
