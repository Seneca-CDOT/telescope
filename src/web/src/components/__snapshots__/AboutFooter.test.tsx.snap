// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root makeStyles-root-1 MuiGrid-container"
  >
    <div
      class="MuiBox-root MuiBox-root-8"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-flex-start MuiGrid-justify-xs-space-between"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
        >
          <img
            alt="Telescope Logo"
            height="60"
            src="/logo.svg"
            width="60"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-6"
          >
            <h5
              class="MuiTypography-root makeStyles-heading-2 MuiTypography-h5"
            >
              DOCS
            </h5>
            <hr
              class="MuiDivider-root makeStyles-leftDivider-3"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              <a
                class="makeStyles-links-5"
                href="https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md"
              >
                Get Started
              </a>
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              <a
                class="makeStyles-links-5"
                href="https://github.com/Seneca-CDOT/telescope/blob/master/docs/CONTRIBUTING.md"
              >
                Contribute
              </a>
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-flex-end MuiGrid-grid-xs-6"
          >
            <h5
              class="MuiTypography-root makeStyles-heading-2 MuiTypography-h5"
            >
              MORE
            </h5>
            <hr
              class="MuiDivider-root makeStyles-rightDivider-4"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              <a
                class="makeStyles-links-5"
                href="https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List"
              >
                Planet CDOT Feed List
              </a>
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-6"
        >
          <h5
            class="MuiTypography-root makeStyles-heading-2 MuiTypography-h5"
          >
            COMMUNITY
          </h5>
          <hr
            class="MuiDivider-root makeStyles-leftDivider-3"
          />
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
          >
            <div
              class="MuiGrid-root MuiGrid-item"
            >
              <a
                class="makeStyles-links-5"
                href="https://github.com/Seneca-CDOT/telescope"
              >
                 
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                  />
                </svg>
              </a>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item"
            >
              <a
                class="makeStyles-links-5"
                href="https://seneca-open-source.slack.com/archives/CS5DGCAE5"
              >
                <svg
                  fill="currentColor"
                  height="22"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 448 512"
                  width="22"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root makeStyles-card-7 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-4"
      >
        <div>
          GitHubContributorCard
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h6
        class="MuiTypography-root makeStyles-footer-6 MuiTypography-h6"
      >
        Copyright © 2021 Seneca’s Centre for Development of Open Technology
      </h6>
    </div>
  </div>
</DocumentFragment>
`;
